test_util/aws.py

```diff
@@ -99,6 +97,18 @@ def create_stack(self, name, template_url, parameters, deploy_timeout=60):
             # so that we may pass parameters directly from yaml (which parses numbers as non-strings)
             Parameters=param_dict_to_aws_format(parameters))
 
+    def create_stack_from_string(self, name, template, parameters, deploy_timeout=60):
```

Comment:
This duplicates a lot of `create_stack()`. IMO it'd make more sense to add a `template` arg to `create_stack()`, since that better reflects what the actual CloudFormation API looks like.

```
def create_stack(self, name, parameters, template_url=None, template=None, deploy_timeout=60):
    ...
```

I _think_ you can pass all those params straight through to boto's `create_stack()` and it'll validate that you passed exactly one of the template args.